rllib/utils/metrics/tests/test_stats.py

```diff
@@ -184,7 +185,7 @@ def test_reduce_with_clear():
     check(len(stats), 0)  # Stats should be cleared
 
 
-def test_merge_on_time_axis():
+def test_basic_merge_on_time_axis():
```

Comment:
Similar in spirit to the renaming from test_metrics_logger.py, these are now the basic test while merging is tested in depth elsewhere.